frontend/src/properties/propertysections/TreePropertySection.tsx

```diff
@@ -0,0 +1,100 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+
+import { makeStyles } from '@material-ui/core/styles';
+import { ClassNameMap } from '@material-ui/core/styles/withStyles';
+import Typography from '@material-ui/core/Typography';
+import ChevronRightIcon from '@material-ui/icons/ChevronRight';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import { TreeItem } from '@material-ui/lab';
+import TreeView from '@material-ui/lab/TreeView';
+import { httpOrigin } from 'common/URL';
```

Comment:
Do not use `httpOrigin` directly, rely on `useContext(ServerContext)`. Long story short, with this code your component would only work in an environment where the dev/prod use cases are the same as in Sirius Web. For example, this code cannot work in the VS Code extension.